doc/api/events.md

```diff
@@ -307,6 +307,43 @@ to each.
 
 Returns `true` if the event had listeners, `false` otherwise.
 
+
+```js
+const EventEmitter = require('events');
+const myEmitter = new EventEmitter();
+
+// First listener
+myEmitter.on('event', function firstListener() {
+  console.log('Helloooo! first listener');
+});
+// Second listener
+myEmitter.on('event', function secondListener(arg1, arg2) {
+  console.log(`an event with parameters ${arg1}, ${arg2}, occurred in second listener`);
+});
+// Third listener
+myEmitter.on('event', function thirdListener(...args) {
+  let parameters = '';
+  args.forEach((arg) => {
+    parameters += `${arg}, `;
+  }
+  );
+  console.log(`an event with parameters ${parameters}occurred in third listener`);
+});
+
+console.log(myEmitter.listeners('event'));
+
+myEmitter.emit('event', 1, 2, 3, 4, 5);
+
+// Prints:
+// [ [Function: firstListener],
+// [Function: secondListener],
+// [Function: thirdListener] ]
```

Comment:
Using current `node` from the master branch, the formatting is slightly different:

```console
[
  [Function: firstListener],
  [Function: secondListener],
  [Function: thirdListener]
]
Helloooo! first listener
an event with parameters 1, 2, occurred in second listener
an event with parameters 1, 2, 3, 4, 5, occurred in third listener
```

Reply:
ok sure